lib/sigs/delegated/eth.go

```diff
@@ -0,0 +1,26 @@
+package delegated
+
+import (
+	"fmt"
+
+	"golang.org/x/crypto/sha3"
+)
+
+// EthAddressFromPubKey returns the Ethereum address corresponding to an
+// uncompressed secp256k1 public key.
+//
+// TODO move somewhere else, this likely doesn't belong here.
+func EthAddressFromPubKey(pubk []byte) ([]byte, error) {
```

Comment:
Assuming it doesn't cause import cycles, the eth types place might be a good home.

Reply:
but honestly i'm...fine with it being here.